src/arch/xtensa/Kconfig

```diff
@@ -17,3 +17,9 @@ config WAKEUP_HOOK
 	  This config should be selected by other platform-level configs.
 	  Platforms that use it, have to implement hook function
 	  platform_interrupt_on_wakeup.
+
+config INLINE_CONSIDER_ALL_FUNCTIONS
+        bool
+        default n
+        help
+          Indicate whether noinine_functions are added into build option.
```

Comment:
I'd describe exactly what this does: "When enabled, -fno-inline-function option is not passed to compiler"